Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -16,25 +16,6 @@
         qTest.cleaningThresholds.append(30.)
         qTest.flags.append('HFDigi')
              
-particleFlowRecHitHF.producers[0].qualityTests.append(
```

Comment:
Slava, 
I take the liberty to include Michalis in the thread.
He has asked me to remove only HF timing-based selection fragment and it's 
supposed to slightly change both Run 1/2 results.

   Salavat


On Wed, 30 Sep 2015, Slava Krutelyov wrote:

> 
> In Configuration/StandardSequences/python/Reconstruction_Data_cff.py:
> 
> > @@ -16,25 +16,6 @@
> >          qTest.cleaningThresholds.append(30.)
> >          qTest.flags.append('HFDigi')
> > 
> > -particleFlowRecHitHF.producers[0].qualityTests.append(
> 
> for some reason I thought that this change was not supposed to change anything as it was redundant.
> I see small changes in HF-related objects in both run1 and run2 data.
> @bachtis could you please clarify
> Thank you.
> 
> —
> Reply to this email directly or view it on GitHub.[AEx02pi3qIEoxDrx-ixRaiH4JwpFmBQbks5o3EDGgaJpZM4GGgk3.gif]
> 
> 
>